### ISSUE TITLE
Multi-module build with nested references fails

### ISSUE THREAD

**Issue by de55** [GoogleContainerTools/jib#2285]
**Environment**:

Jib version: 2.0.0
Build tool: Gradle 6.1.1
OS: ubuntu:18.04

**Description of the issue**:
Jib fails whenever there are nested project dependencies. i.e. if there is a project dependency structure such that: A depends on B and B depends on C. The build for project A will fail because of project C.

**Expected behavior**:
Jib successfully builds

**Steps to reproduce**:

  1. Create multimodule gradle project
  2. Create project A, B, C
  3. Have project A depend on project B
  4. Have project B depend on project C
  5. Try to build a docker file using :A:jibDockerBuild

**Log output**: 
Caused by: java.nio.file.NoSuchFileException: /builds/xxx/xxx.jar
 	at com.google.cloud.tools.jib.api.JavaContainerBuilder.addProjectDependencies(JavaContainerBuilder.java:359)
 	at com.google.cloud.tools.jib.gradle.GradleProjectProperties.createJibContainerBuilder(GradleProjectProperties.java:211)
 	... 194 more

**Comment by loosebazooka** [comment]
hrmm... this could be due to the `java-library` plugin issue. Fixed in: https://github.com/GoogleContainerTools/jib/pull/2247

However, just to confirm. Can you:
1. Confirm whether C is using the `java-library` plugin
1. Run and show us the output of
    ```
    ./gradlew :A:jibDockerBuild --dry-run --console=plain
    ```

**Comment by de55** [comment]
Yes I am using ```java-library```, and here is the output:

```
$ gradle :A:jibDockerBuild --dry-run --console=plain
:C:compileKotlin SKIPPED
:C:compileJava SKIPPED
:B:compileKotlin SKIPPED
:B:compileJava SKIPPED
:A:compileKotlin SKIPPED
:A:compileJava SKIPPED
:A:processResources SKIPPED
:A:classes SKIPPED
:B:processResources SKIPPED
:B:classes SKIPPED
:B:inspectClassesForKotlinIC SKIPPED
:B:jar SKIPPED
:B:assemble SKIPPED
:A:jibDockerBuild SKIPPED
```

**Comment by loosebazooka** [comment]
okay, so I believe this is fixed by #2247. depending on `runtimeClasspath` should fix this issue. If you want a temporary fix while you wait on a `2.1.0` release, you can do:

(build.gradle)
```
jibDockerBuild.dependsOn sourceSets.main.runtimeClasspath
```

Or you can even test the fix in 2.1.0, by cloning the repo and building a local copy of the plugin (https://github.com/GoogleContainerTools/jib/blob/master/CONTRIBUTING.md#debugging-the-jib-gradle-plugin-jib-gradle-plugin)

**Comment by de55** [comment]
That worked. Thanks.

**Comment by chanseokoh** [comment]
@de55 we've released Jib 2.1.0 with the fix.
